src/cpu/cpu.cpp

```diff
@@ -3078,21 +3078,26 @@ class Cpu final {
 		} else if (cpu_type == "386_prefetch") {
 			CPU_ArchitectureType = ArchitectureType::Intel386Fast;
 
-			if (cpu_core == "normal") {
-				cpudecoder = &CPU_Core_Prefetch_Run;
-
-				CPU_PrefetchQueueSize = 16;
-
-			} else if (cpu_core == "auto") {
+			if (cpu_core == "auto") {
 				cpudecoder = &CPU_Core_Prefetch_Run;
 
 				CPU_PrefetchQueueSize         = 16;
 				auto_determine_mode.auto_core = false;
```

Comment:
If CPU core is auto, then auto_core is false.  Is that intentional?

Reply:
Yeah, intentional. I've just flipped this around:

<img width="684" alt="image" src="https://github.com/dosbox-staging/dosbox-staging/assets/698770/d4b00b13-82d4-4c82-be5f-d6a330972a36">

I agree with your point that it's confusing. Well, the whole CPU code is confusing, and although I tried to improve it with better namings and stuff, it's still incredibly tangled and I'm very reluctant to change the logic substantially.